### DISCUSSION TITLE
1 Project, many Tasks, Task is dependent from other Task

### DISCUSSION THREAD

**Discussion by Bernie1502** [Budibase/budibase#5741] [Help]
Hi,
i want to create a app to order Tasks and Works.

I have a (mySQL-)Table with Tasks:
| Id | Task | Duration
| 1 | Cook Spaghetti | 1
| 2 | Buy Ingredients | 1
| 3 | eat Lunch | 2
| 4 | invite Wife | 1

The Taks are dependent on each other, this i want to realize in another Table "Dependency"
| Id | Task | Before
| 1 | 3 | 1
| 2 | 1 | 2 
| 3 | 3 | 4

I think, it should be so:
![2022-05-06 09_45_04-db drawio - diagrams net und 5 weitere Seiten - Geschäftlich – Microsoft​ Edge](https://user-images.githubusercontent.com/49621100/167089363-5821fc80-6a68-423a-b37e-2180473ef555.png)

But, how do i set the relationship in BB and Display it in the Designs? Or is my concept wrong?

Thanks for any ideas!

**Comment by mjashanks** [comment]
Hi @Bernie1502 - I'm assuming that you have setup the relationship in Budibase. Have you also det the "Display Column" on the tasks table? This tells Budibase which column to display when listing the relationship.

**Reply by Bernie1502** [reply]
Hi @mjashanks,
thanks for your reply. Yes i have setup two "one to many"-relationships in Budibase from Tasks to Dependency:
* id -> Task_Id
* id -> Before_Task_ID
and the Display Column is set on "Task" in the Tasks-Table

Thanks
